spec/spec.go

```diff
@@ -82,6 +82,8 @@ var supportedSchemaFields = []string{
 	// This is currently being used to store additional metadata for our SDKs. It's
 	// passed through our Spec and should be ignored
 	"x-stripeResource",
+	"x-stripeOperations",
+	"x-stripeParam",
```

Comment:
I'm using the SDK beta spec instead of `spec3.yaml`-like spec. I think we need to move to SDK specs across the board but we'll start with beta.